python/README.md

```diff
@@ -66,10 +66,14 @@ It will generate various reports and summaries:
 
 ### Test stats
 
-Current status: ![coverage](http://www.yarntomato.com/percentbarmaker/button.php?barPosition=29&leftFill=) (**1685**/5787 passed)
+Current status: ![coverage](http://www.yarntomato.com/percentbarmaker/button.php?barPosition=29&leftFill=) (**1709**/5787 passed)
```

Comment:
**1709**/5787=0.2953...~=30%, so feel free to increase the percentage ;)

```suggestion
Current status: ![coverage](http://www.yarntomato.com/percentbarmaker/button.php?barPosition=30&leftFill=) (**1709**/5787 passed)
```

Just a thought that maybe we need to find an API that calculates the percentage itself...

Reply:
Do we need this percentage? It's false anyway - once we started, there were around 15% already :D I think it's enough to have the number of passing VS. Failing/all tests. Wdyt?

Reply:
Yeah, I don't mind removing this image

Reply:
https://github.com/krzema12/kotlin-python/pull/32